upgrade/src/app/api/og/route.tsx

```diff
@@ -117,8 +117,8 @@ async function getFont<TWeights extends readonly number[]>({
     .split("@font-face {")
     .splice(1)
     .map((font) => {
-      const u = font.match(/src: url\((.+)\) format\('(opentype|truetype)'\)/);
-      const w = font.match(/font-weight: (\d+)/);
+      const u = /src: url\((.+)\) format\('(opentype|truetype)'\)/.exec(font);
+      const w = /font-weight: (\d+)/.exec(font);
```

Comment:
[`@typescript-eslint/prefer-regexp-exec`](https://v8--typescript-eslint.netlify.app/rules/prefer-regexp-exec) was promoted to the [`stylistic-type-checked` config](https://v8--typescript-eslint.netlify.app/users/configs/#stylistic-type-checked) in v8.